WcaOnRails/app/models/post.rb

```diff
@@ -5,6 +5,21 @@ class Post < ActiveRecord::Base
   validates :body, presence: true
   validates :slug, presence: true, uniqueness: true
 
+  BREAK_TAG_RE = /<!--\s?break\s?-->/
```

Comment:
Yeah 0 or 1. If we want any number of spaces I don't even think we need a non greedy here because it's just spaces and we end with the -->

So BREAK_TAG_RE = /<!--\s*break\s*-->/ should do the trick I think?